Formula/portable-openssl.rb

```diff
@@ -47,57 +46,38 @@ def configure_args
     args = %W[
       --prefix=#{prefix}
       --openssldir=#{openssldir}
-      no-ssl2
-      no-ssl3
       no-shared
-      enable-cms
     ]
 
-    if OS.mac?
-      args << "zlib-dynamic"
-    else
-      args << "-L#{Formula["portable-zlib"].opt_prefix/"lib"}"
-      args << "zlib"
-    end
-
     args
   end
 
   def install
-    # Load zlib from an explicit path instead of relying on dyld's fallback
-    # path, which is empty in a SIP context. This patch will be unnecessary
-    # when we begin building openssl with no-comp to disable TLS compression.
-    # https://langui.sh/2015/11/27/sip-and-dlopen
-    if OS.mac?
-      inreplace "crypto/comp/c_zlib.c",
-                'zlib_dso = DSO_load(NULL, "z", NULL, 0);',
-                'zlib_dso = DSO_load(NULL, "/usr/lib/libz.dylib", NULL, DSO_FLAG_NO_NAME_TRANSLATION);'
-    end
-
     ENV.deparallelize
     system "perl", "./Configure", *(configure_args + arch_args)
-    system "make", "depend"
```

Comment:
Do we still need `makedepend` if this is gone?

Reply:
`make depend` still exists but is now implicitly done.

I'm not entirely if the `makedepend` dependency was needed in the first place. It's not used at all when building OpenSSL and Ruby in homebrew-core. I guess I can remove it and see if anything breaks.

Reply:
Yeh, let's see what happens 👍

Reply:
Looks like it built fine!